doc/naming.md

```diff
@@ -34,15 +34,19 @@ Most gRPC implementations support the following URI schemes:
     resolver does not support this, but the c-ares based resolver
     supports specifying this in the form "IP:port".)
 
-- `unix:path`, `unix://absolute_path`, or `unix-abstract:abstract_path` -- Unix domain sockets (Unix systems only)
+- `unix:path`, `unix://absolute_path` -- Unix domain sockets (Unix systems only)
   - `path` indicates the location of the desired socket.
   - In the first form, the path may be relative or absolute; in the
     second form, the path must be absolute (i.e., there will actually be
     three slashes, two prior to the path and another to begin the
     absolute path).
-  - In the third form, `abstract_path` indicates a name in the abstract namespace:
-     - The name has no connection with filesystem pathnames.
-     - No permissions will apply to the socket - any process/user may access the socket.
+
+- `unix-abstract:abstract_path` -- Unix domain socket in abstract namespace (Unix systems only)
+  - `abstract_path` indicates a name in the abstract namespace:
+  - The name has no connection with filesystem pathnames.
+  - No permissions will apply to the socket - any process/user may access the socket.
+  - Abstract sockets are represented by a null byte ('\0') as the first character;
+    the dialler will prepend this null. Do not include the null in `abstract_path`.
```

Comment:
The concept of a "dialler" is go-specific; we don't really use that term in this document.  Let's just say "implementation" instead of "dialler".

Reply:
Done